src/utility/api.tsx

```diff
@@ -0,0 +1,23 @@
+type Task = {
+    task: {
+        status: string,
+        id: number
+    }
+}
+
+export const sendRequestToApi = async (axiosRequest: any, expectedStatus: number, url = "", body:Task | null = null) => {
```

Comment:
I set the type of axiosRequest: to any cause the types for all the various potential axiosRequest functions that could be passed in had different inherent types. Is there a problem with leaving it as type = `any`? I'm not sure if there is a way to do it the right way, if there is I'd love to figure out what I'm doing wrong.

Reply:
there's no problem, you can leave it as any for cases like this